lci_strategic_plugin/src/lci_strategic_plugin.cpp

```diff
@@ -585,6 +574,17 @@ void LCIStrategicPlugin::planWhenAPPROACHING(const cav_srvs::PlanManeuversReques
 
   ROS_DEBUG_STREAM("intersection_end_downtrack_: " << intersection_end_downtrack_.get());
 
+  // If the vehicle is at a stop trigger the stopped state
+  constexpr double HALF_MPH_IN_MPS = 0.22352;
+  if (current_state.speed < HALF_MPH_IN_MPS &&
+      fabs(distance_remaining_to_traffic_light) < config_.stopping_location_buffer + length_to_front_bumper_)
```

Comment:
Seems like a good opportunity to remove the length_to_front_bumper from this check. To get the same behavior is before you could correspondingly increase stopping_location_buffer parameter value.